modules/fts/pages/fts-quick-index.adoc

```diff
@@ -0,0 +1,61 @@
+= Quick Index
+
+To create a quick index, left-click on the QUICK INDEX button, towards the right-hand side:
+
+image::fts-quick-index-button.png[,300,align=left]
+
+The QUICK INDEX screen appears:
+
+image::fts-quick-index-screen.png[,300,align=left]
+
+To define a basic index on which Full Text Search can be performed, begin by entering a unique name for the index into the Index Name field, at the upper-left: for example, travel-sample-index. (Note that only alphanumeric characters, hyphens, and underscores are allowed for index names. Note also that the first character of the name must be an alphabetic character.) Then, use the pull-down menu provided for the Keyspace field, at the upper-right, to specify as follows:
+
+bucket: `travel-sample`
+
+scope: `inventory`
+
+collection: `hotel`
+
+image::fts-quick-index-name-and-bucket.png[,300,align=left]
+
+The user can continue to randomly pick documents until they find a document of their intended type/schema. It is also possible to have multi-schema documents within a collection.
+
+image::fts-quick-index-json.png[,300,align=left]
+
+Select the required field from the document, which is needed to be mapped to this index. Once the field is selected, the configuration panel is displayed at the right.
+
+image::fts-quick-index-json-configuration.png[,300,align=left]
+
+Select the related type for the field from the dropdown. Select Index this field as an identifier to index the identifier values exactly as it is without any transformation; for this case, language selection is disabled. 
+
+Configuration allows you to specify the language selection for the specified field. You can select the given configuration , but it requires additional storage and will make the index larger.
+
+Include In search results, Support phrase matching, and Support sorting and faceting. Searchable As field allows you to modify searchable input for the selected field.
+
+image::fts-quick-index-searchable-input.png[,300,align=left]
+
+Once the configuration is completed for the selected fields, click Add. Mapped fields will display the updated columns.
+
+image::fts-quick-index-json-mapping.png[,300,align=left]
+
+This is all you need to specify in order to create a basic index for test and development. No further configuration is required. 
+
+Note, however, that such default indexing is not recommended for production environments since it creates indexes that may be unnecessarily large, and therefore insufficiently performant. To review the wide range of available options for creating indexes appropriate for production environments, see Creating Indexes.
```

Comment:
Following this example, what is wrong with this index mapping?  It feels like this warning was copy/pasted from previous example which created a default index mapping (which is wasteful)

Reply:
Martin, could you please provide some more details here? I guess you want to say that the note is not relevant to this topic.

Reply:
No, it is not a value that you just blindly set to something else.  You would need a whole section or page talking about types.  Only in that context does it make sense to change it, as you'd have to set up a lot of other things before you can change it to anything meaningful.

Reply:
Update: Remove the note.